python/src/aiconfig/Config.py

```diff
@@ -381,20 +397,41 @@ def save(self, json_config_filepath: str | None = None, include_outputs: bool =
         if not include_outputs:
             exclude_options["prompts"] = {"__all__": {"outputs"}}
 
-        if not json_config_filepath:
-            json_config_filepath = self.file_path or "aiconfig.json"
-
-        with open(json_config_filepath, "w") as file:
-            # Serialize the AI Configuration to JSON and save it to the file
-            json.dump(
-                self.model_dump(
-                    mode="json",
-                    exclude=exclude_options,
-                    exclude_none=True,
-                ),
-                file,
-                indent=2,
+        default_filepath = "aiconfig.json" if mode == "json" or mode is None else "aiconfig.yaml"
```

Comment:
nit, checking yaml and defaulting to json -- `default_filepath = "aiconfig.yaml" if mode == "yaml" else "aiconfig.json"` -- is probably a better reflection of intention

But also, should this default be something like `Untitled.aiconfig.json` / `Untitled.aiconfig.yaml` to reflect aiconfig as part of the file extension now?